src/main/webapp/completedPath.js

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 Google LLC
```

Comment:
This file has a lot of the same functions as gameDialogue.js. Could you potentially refactor and create a common helper file with these common functions and export the functions that you need?  https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export

Reply:
+1

Reply:
+1

Reply:
I will include this in the next PR to keep the PR short and easy to read

Reply:
Please do the refactor in this CL--> Keeping the CL short isn't as important as doing it right